managed/services/victoriametrics/scrape_configs.go

```diff
@@ -590,5 +590,23 @@ func scrapeConfigsForVMAgent(s *models.MetricsResolutions, params *scrapeConfigP
 			Labels:  labels,
 		}},
 	}
-	return []*config.ScrapeConfig{cfg}, nil
+	return []*config.ScrapeConfig{cfg, scrapeConfigForPmmAgent(s, params)}, nil
```

Comment:
probably `scrapeConfigForPmmAgent` shouldn't be called from here. let's call it from the place where `scrapeConfigsForVMAgent` is called.